JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -91,185 +99,274 @@
                 </owl:unionOf>
             </owl:Class>
         </rdfs:range>
-        <rdfs:isDefinedBy>http://www.ontology-of-units-of-measure.org/resource/om-2/</rdfs:isDefinedBy>
-        <rdfs:label>has unit</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://www.ontology-of-units-of-measure.org/resource/om-2/"/>
```

Comment:
Could you double check if this syntax is correct - it might be that the TBox generator did not produce the correct results

Reply:
George mentioned that the current TBoxGeneration class in JPS_BASE_LIB handles this correctly.